src/LiFi.unit.spec.ts

```diff
@@ -211,4 +269,49 @@ describe('LIFI SDK', () => {
       )
     })
   })
+
+  describe('Should pick up gas from signer estimation', () => {
+    it('should pick up gas estimation from signer', async () => {
+      const route = buildRouteObject({
+        step,
+      })
+
+      await lifi.executeRoute(signer, route)
+
+      expect(signer.sendTransaction).toHaveBeenCalledWith({
+        gasLimit: '125000',
+        // TODO: Check the cause for gasLimit being outside transactionRequest. Currently working as expected in widget
```

Comment:
Do you want to fix that in this PR or in another one?

If you want to leave that as TODO here, please mention it in the PR description

Reply:
Duplicate. Resolving this